app/views/games/categories/_title.slim

```diff
@@ -2,24 +2,24 @@
   .row.no-gutters
     - if @game.srdc&.cover_url
       .col-md-4 style='max-width: 200px'
-        img.card-img src=@game.srdc.cover_url
+        img.card-img src=@game.srdc.cover_url alt="#{@game.srdc.name}'s cover"
     .col-md-8
       .card-body
         h1.card-title = @category.game
         h6
           .btn-group.mr-2
             - if @on_game_page && @category.game.srdc.try(:url).present?
-              a.btn.btn-dark.tip href=@category.game.srdc.url title='See on Speedrun.com'
-                = image_tag(asset_path('srdc.png'), style: 'height: 0.8em')
+              a.btn.btn-dark.tip href=@category.game.srdc.url title='See on Speedrun.com' aria-label='Veiw on Speedrun.com'
+                = image_tag(asset_path('srdc.png'), style: 'height: 0.8em', alt: 'Speedrun.com first place trophy')
             - elsif !@on_game_page && @category.srdc.try(:url).present?
-              a.btn.btn-dark.tip href=@category.srdc.url title='See on Speedrun.com'
+              a.btn.btn-dark.tip href=@category.srdc.url title='See on Speedrun.com' aria-label='View on Speedrun.com'
```

Comment:
I noticed this month that speedrun.com posts their preferred capitalization as "speedrun.com", so when I add or edit uses I've been lazily updating them: https://www.speedrun.com/rules (5th bullet point)

Although for the beginnings of sentences definitely still capitalize it as you would any other word.